README.md

```diff
@@ -2,6 +2,17 @@
 
 This is the content repository for [jamulus.io](https://jamulus.io). Text appearing in the Jamulus software application itself is part of the [main code repository](https://github.com/jamulussoftware/jamulus). See also the [general overview](https://jamulus.io/wiki/Contribution) to getting involved. 
 
+## Translation
+
+<a href="https://hosted.weblate.org/engage/jamulus/">
+<img src="https://hosted.weblate.org/widgets/jamulus/-/multi-auto.svg" alt="Translation status" />
+</a>
+The Jamulus website is translated using [Weblate](https://hosted.weblate.org/engage/jamulus/). If you prefer you may also submit translations via Pull Requests. Note that translating the Jamulus application itself is done by a different process, but you are welcome to help with that too. Introduce yourself on the [GitHub Discussion](https://github.com/jamulussoftware/jamulus/discussions) and say what you’d like to do.
+
+#### Please also note that discussions should be in English so as to be open to all.
+
+For translating text in the Jamulus application itself, please see [TRANSLATING.md](https://github.com/jamulussoftware/jamulus/blob/master/docs/TRANSLATING.md) in the application repository.
```

Comment:
Why does the para at line 10 talk about the app translations if we have this para?  It might be easier to follow if there was only one para about app translations.  Given line 10 says "we don't do app translations here", this para could probably just be dropped.

Reply:
Yes. A lot of the README reads like it's been cobbled together like a patchwork over time (it has). I'll give it a more thorough read to try and iron out some creases.